examples/jacobi/src/gpu/kernels/sycl.cmake

```diff
@@ -15,15 +15,14 @@ target_compile_definitions(${TARGET_NAME} PRIVATE DEVICE_${DEVICE_BACKEND}_LANG
 target_include_directories(${TARGET_NAME} PUBLIC root)
 target_link_libraries(${TARGET_NAME} PUBLIC device)
 
-if (${DEVICE_BACKEND} STREQUAL "hipsycl")
-    if(NOT DEFINED HIPSYCL_TARGETS)
-        set(HIPSYCL_TARGETS "cuda:sm_${DEVICE_ARCH}")
-    endif()
-
-    find_package(hipSYCL CONFIG REQUIRED)
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/root/cmake" "${CMAKE_ROOT}/Modules")
```

Comment:
Just in case some user may use some custom `CMAKE_MODULE_PATH`... Maybe replace by

`set( CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/root/cmake" ${CMAKE_MODULE_PATH} )`  ?

Or, by https://stackoverflow.com/a/52730517 , it could also be

`list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/root/cmake")`

?